.github/workflows/test.yml

```diff
@@ -52,4 +52,11 @@ jobs:
         projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
         zip: true
         exitZeroOnChanges: true
-        onlyChanged: true
\ No newline at end of file
+        onlyChanged: true
+    - name: Storybook test
```

Comment:
@martha I am wondering about the choice to user storybook test runner on CI as opposed to running tests via Chromatic on CI. The storybook gives the [example](https://storybook.js.org/docs/writing-tests/test-runner#whats-the-difference-between-chromatic-and-test-runner) that you might want to "Use [test runner] locally and Chromatic on your CI". I would guess that a benefit of Chromatic would be if the tests fail, maybe you can see some more detailed output on chromatic as opposed to just test logs. (Judging from [chromatic docs about PR workflow](https://www.chromatic.com/docs/in-pull-request/) it seems pretty advanced).

Linking https://github.com/open-path/Green-River/issues/6687 for reference too

Reply:
Sorry, I didn't read all the way through your description 🙈 

> I believe, alternatively, it's possible to run tests in CI against a Chromatic instance (see [this other recipe](https://storybook.js.org/docs/writing-tests/test-runner#run-against-deployed-storybooks-via-github-actions-deployment)), but based on the existing "Run Chromatic" build step, it looks like we only want to run chromatic on push to a release branch and not on pull request.

That was set up mostly for the purpose of publishing the "release version" of the storybook. I don't have anything against using chromatic to run testing on PRs. (And I guess publishing, maybe that's the norm for chromatic usage).

Reply:
Sounds good, I updated the PR to run the tests against the Chromatic instance of Storybook.

We do still need the test runner to run the accessibility tests. Chromatic supports running interaction tests, but [doesn't support a11y tests](https://github.com/storybookjs/storybook/discussions/17966) as far as I can tell. You can see this in the example build [here](https://github.com/greenriver/hmis-frontend/actions/runs/11223801489/job/31199218726?pr=935). The test runner step shows a11y failures, but the Chromatic step succeeds. In the corresponding [Chromatic build](https://www.chromatic.com/build?appId=66e877f06b978106d75d04bc&number=34) for that action, there are 51 "unreviewed changes" (bc I turned on the "visual tests" setting) but there are no a11y-related failures or warnings.